<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>linacGui</class>
 <widget class="TaurusFrame" name="linacGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1216</width>
    <height>935</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="linacTabs">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <widget class="QWidget" name="communicationTab">
      <attribute name="title">
       <string>Communication</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1186</width>
            <height>884</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <spacer name="horizontalSpacer_left">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2">
            <widget class="linacPlcWidget" name="plc1">
             <property name="model">
              <string>li/ct/plc1</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="linacOverview" name="linacOverview"/>
           </item>
           <item row="2" column="2">
            <widget class="linacPlcWidget" name="plc4">
             <property name="model">
              <string>li/ct/plc4</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="linacPlcWidget" name="plc5">
             <property name="model">
              <string>li/ct/plc5</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="linacPlcWidget" name="plc2">
             <property name="model">
              <string>li/ct/plc2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <spacer name="horizontalSpacer_right">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="1">
            <widget class="linacPlcWidget" name="plc3">
             <property name="model">
              <string>li/ct/plc3</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <spacer name="verticalSpacer_down">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2">
            <spacer name="verticalSpacer_up">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="startupTab">
      <attribute name="title">
       <string>Start up</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="mainscreenTab">
      <attribute name="title">
       <string>Main Screen</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaurusFrame</class>
   <extends>QFrame</extends>
   <header>taurus.qt.qtgui.container</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TaurusWidget</class>
   <extends>QWidget</extends>
   <header>taurus.qt.qtgui.container</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>linacOverview</class>
   <extends>TaurusWidget</extends>
   <header>linacoverview</header>
  </customwidget>
  <customwidget>
   <class>linacPlcWidget</class>
   <extends>TaurusWidget</extends>
   <header>linacplcwidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
